pkg/lib/serverTime.js

```diff
@@ -562,26 +564,17 @@ function ChangeSystimeBody({ state, errors, change }) {
     }
 
     const ntp_servers = (
-        <table>
-            <tbody>
-                { custom_ntp.servers.map((s, i) => (
-                    <tr key={i}>
-                        <td style={{ width: "100%" }}>
-                            <input type="text" className="form-control" value={s} placeholder={_("NTP server")}
-                onChange={event => change_server(event, i, event.target.value)} />
-                        </td>
-                        <td>
-                            <Button variant="secondary" onClick={event => add_server(event, i)} icon={ <PlusIcon /> } />
-                        </td>
-                        <td>
-                            <Button variant="secondary" onClick={event => remove_server(event, i)}
-                                    icon={ <CloseIcon /> }
-                                    isDisabled={i == custom_ntp.servers.length - 1} />
-                        </td>
-                    </tr>))
-                }
-            </tbody>
-        </table>);
+        custom_ntp.servers.map((s, i) => (
+            <InputGroup className="ntp-server-input-group" key={i}>
+                <TextInput value={s} placeholder={_("NTP server")} aria-label={_("NTP server")}
+                           onChange={(value, event) => change_server(event, i, value)} />
+                <Button variant="secondary" onClick={event => add_server(event, i)}
+                        icon={ <PlusIcon /> } />
+                <Button variant="secondary" onClick={event => remove_server(event, i)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18508-20230315-131044-159e27b8-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)